spec/schema.rb

```diff
@@ -1,14 +1,17 @@
 ActiveRecord::Schema.define :version => 0 do
   create_table :flaggable_models, :force => true do |t|
     t.string :name
+    t.integer :flaggings_count
```

Comment:
I added them by hand. They would have become necessary after the counter cache was added, but no one updated the specs.